pkgs/by-name/he/heimdall-udev-rules/package.nix

```diff
@@ -0,0 +1,24 @@
+{
+  heimdall,
+  lib,
+  runCommandLocal,
+}:
+
+runCommandLocal "heimdall-udev-rules"
```

Comment:
If I'm looking at the build output correctly the derivation name is currently not getting versioned. I somewhat doubt that that is intentional, but I'm also not sure if it's a requirement in the first place.

Just wanted to point it out, feel free to mark as resolved without further comment if having this non-versioned is totally fine.

Reply:
added udev output instead of heimdall-udev-rules